frontend/src/core/comments/components/Comment.css

```diff
@@ -0,0 +1,65 @@
+.history-comment {
```

Comment:
We are likely to use this component for the Team Comments as well, so `history-comment` is not the best name. I'd suggest you add a `className` on the outer `div` element in the `CommentsList` (e.g. `comment-list`) and then change this selector to `.comment-list comment`.